src/start_chroot_script

```diff
@@ -14,7 +14,16 @@ source /common.sh
 install_cleanup_trap
 
 if [[ "${EDITBASE_DISTRO}" == "armbian" || "${EDITBASE_DISTRO}" == "ubuntu" ]]; then
-  mv /etc/resolv.conf /etc/resolv.conf.orig || true
+  # workaround for Armbian > 24.5, because it uses a symlink for /etc/resolv.conf
+  if [ -h /etc/resolv.conf ]; then
+    link_target="$(readlink /etc/resolv.conf.orig)"
+    echo "${link_target}" > /etc/resolv.conf.link
```

Comment:
Shouldn't this be `/etc/resolv.conf`? The `orig` version doesn't yet exist at this point.

Reply:
yep! you're absolutely right! i must have made a mistake when copy&paste from my test branch. i fixed it.